server/engine/Game.js

```diff
@@ -21,9 +21,54 @@ class Game {
 		this.numGuessesLeft = 0;
 		this.spyHint = "";
 		this.madeGuess = false;
+		this.hostID = "";
+		this.redSpy = "";
+		this.blueSpy = "";
+		this.redField = "";
+		this.blueField = "";
 		this.reset();
 	}
 
+	setHost(id) {
+		this.hostID = "";

```

Comment:
should this be passing the id argument into this.hostID?

Reply:
Yes...though I don't believe I'm using it just yet.